### ISSUE TITLE
Slow control interface can't find variable list

### ISSUE THREAD

**Issue by tunnell** [XENON1T/hax#65]
```
from hax import slow_control
slow_control.init_sc_interface()
```

Gives:

KeyError: 'sc_variable_list'

**Comment by tunnell** [comment]
We have to initialize all of hax to use SC routines?  This is just sort of nasty since it means we can't use the slow control module as standalone (one perk of previous implementation).  Humm...

**Comment by JelleAalbers** [comment]
Yes, that's a price we pay for having common options handling. Normally hax.init calls init_sc_interface for you though, so that's still just one line of code to init it (and if speed is an issue, try `hax.init(use_runs_db=False)`)

**Comment by JelleAalbers** [comment]
If you want to use the slow control interface standalone, you can also do:

`hax.config['sc_variable_list'] = hax.hax_dir + '/sc_variables.csv'` 

instead of hax.init(). Close?
